packages/@react-native-windows/cli/src/index.ts

```diff
@@ -59,23 +88,64 @@ export async function generateWindows(
   ns: string,
   options: GenerateOptions,
 ) {
-  if (!fs.existsSync(projectDir)) {
-    fs.mkdirSync(projectDir);
-  }
+  let error;
+  try {
+    if (!fs.existsSync(projectDir)) {
+      fs.mkdirSync(projectDir);
+    }
+
+    installDependencies(options);
+
+    const rnwPackage = path.dirname(
+      require.resolve('react-native-windows/package.json', {
+        paths: [projectDir],
+      }),
+    );
+    const templateRoot = path.join(rnwPackage, 'template');
+    await copyProjectTemplateAndReplace(
+      templateRoot,
+      projectDir,
+      name,
+      ns,
+      options,
+    );
+  } catch (e) {
+    error = e;
+    telClient.trackException({ exception: error });
+    throw e;
+  } finally {
+    if (!options.noTelemetry && !process.env.AGENT_NAME) {
+      const cwd = process.cwd();
+      const pkgJsonPath = findUp.sync('package.json', {cwd});
+      let rnVersion = '';
+      if (pkgJsonPath) {
+        const pkgJson = require(pkgJsonPath);
+        // check how react-native is installed
+        if (
+          'dependencies' in pkgJson &&
+          'react-native' in pkgJson.dependencies
+        ) {
+          // regular dependency (probably an app), inject into json and run install
+          rnVersion = pkgJson.dependencies['react-native'];
+        }
+      }
 
-  installDependencies(options);
-
-  const rnwPackage = path.dirname(
-    require.resolve('react-native-windows/package.json', {paths: [projectDir]}),
-  );
-  const templateRoot = path.join(rnwPackage, 'template');
-  await copyProjectTemplateAndReplace(
-    templateRoot,
-    projectDir,
-    name,
-    ns,
-    options,
-  );
+      const rnwPkgJson = require('../package.json');
```

Comment:
There's a many:1 relationship between CLI version and RNW version. I.e. they are usually not the same.

Reply:
ah yeah this should capture the CLI version not the RNW version